pages/linux/limactl.md

```diff
@@ -0,0 +1,29 @@
+# limactl
+
+> Virtual machine manager for Linux guests, with multiple VM templates available.
+> Can be used to run containers on macOS, but also for generic virtual machine use cases on macOS and Linux hosts.
+> More information: <https://github.com/lima-vm/lima>.
+
+- Pick any of the available distributions, see templates in `/usr/local/share/lima/templates/`:
+
+`LIMA_DISTRO={{debian|fedora|ubuntu|…}}`
+
+- Create a virtual machine using the default settings (this will download the VM image on the first run):
+
+`cat /usr/local/share/lima/templates/$LIMA_DISTRO.yaml | limactl create --name=$LIMA_DISTRO-vm -`
```

Comment:
```suggestion
- Create a virtual machine using the default settings (this will open a TUI to change the template and other settings):

`limactl create --name {{vm_name}}`
```